components/raftstore/src/store/snap/io.rs

```diff
@@ -176,6 +197,21 @@ where
     };
 
     let instant = Instant::now();
+    let _io_type_guard = WithIoType::new(IoType::Export);
```

Comment:
How about creating a new `IoType` in case snapshot generations interfere with other 'real' `Export` jobs?

Reply:
Good suggestion! It looks like IoType::LoadBalance and IoType::Replication can by used here. so there's no need to create a new IoType.
https://github.com/tikv/tikv/blob/261458672ab66e6acef5017535783628e62bb76c/src/server/snap.rs#L319C1-L324C54